app/screens/reschedule_draft/reschedule_draft.tsx

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useRef, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {Keyboard, SafeAreaView, StyleSheet, View} from 'react-native';
+
+import {updateScheduledPost} from '@actions/remote/scheduled_post';
+import Loading from '@components/loading';
+import {MESSAGE_TYPE, SNACK_BAR_TYPE} from '@constants/snack_bar';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import useAndroidHardwareBackHandler from '@hooks/android_back_handler';
+import useNavButtonPressed from '@hooks/navigation_button_pressed';
+import {usePreventDoubleTap} from '@hooks/utils';
+import DateTimeSelector from '@screens/custom_status_clear_after/components/date_time_selector';
+import {buildNavigationButton, dismissModal, setButtons} from '@screens/navigation';
+import {logDebug} from '@utils/log';
+import {showSnackBar} from '@utils/snack_bar';
+import {changeOpacity} from '@utils/theme';
+import {getTimezone} from '@utils/user';
+
+import type ScheduledPostModel from '@typings/database/models/servers/scheduled_post';
+import type {AvailableScreens} from '@typings/screens/navigation';
+import type {Moment} from 'moment-timezone';
+
+type Props = {
+    currentUserTimezone?: UserTimezone | null;
+    componentId: AvailableScreens;
+    closeButtonId: string;
+    draft: ScheduledPostModel;
+}
+
+const OPTIONS_PADDING = 12;
```

Comment:
this is only used in the style and just once, do we really need to define the constant?

Reply:
Done.